src/test/java/com/google/api/codegen/protoannotations/testdata/library.proto

```diff
@@ -810,6 +809,7 @@ message GetBookFromAnywhereRequest {
   // An alternate book name, used to test restricting flattened field to a
   // single resource name type in a oneof.
   string alt_book_name = 2 [
+    (google.api.field_behavior) = REQUIRED,
```

Comment:
I don't have enough knowledge about the protoannotation migration work so probably have someone else review this part?

Reply:
@lukesneeringer

Reply:
Assuming it is actually a required field (the API would send INVALID_ARGUMENT if it was unset, or something similar) this seems reasonable.

Reply:
Yep, this is to agree with the corresponding library_gapic.yaml, which marks this field as required (see [here](https://github.com/googleapis/gapic-generator/blob/e72072aea215d87b00e6c931b869d53a4e062f11/src/test/java/com/google/api/codegen/testsrc/libraryproto/library_gapic.yaml#L458))